src/components/Button/OvalButton.js

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import { styled } from "@mui/material/styles";
+import { Button } from "@mui/material";
+import { DEFAULT_BUTTON_HEIGHT } from "../../contants/sizes";
+
+const OvalButtonStyled = styled(Button)(({ theme, color }) => {
+  return `
+  background: linear-gradient(45deg, ${theme.palette[color].alternative}, ${theme.palette[color].main});
+  border-radius: 25px;
+  height: ${DEFAULT_BUTTON_HEIGHT};
+  font-size: 17px;
```

Comment:
I wonder if it would make sense to calculate the `border-radius` based on the `DEFAULT_BUTTON_HEIGHT`
```css
border-radius: calc(${DEFAULT_BUTTON_HEIGHT}/2)
```

Reply:
This is actually a good idea, the only issue here is if the button receives a custom height. I am checking if there is a custom height and I use it to calculate the border, if not I am using the default.
I will include it in the next PR.